docs/containers.config.5.md

```diff
@@ -0,0 +1,88 @@
+% containers.conf(5) Container runtime tools configuration file
+
+# NAME
+containers.conf - configuration file of the container runtime tools like podman & buildah
+
+# DESCRIPTION
+The containers configuration file specifies all of the available configuration options and command-line flags for container runtime tools like podman & buildah, but in a TOML format that can be easily modified and versioned.
+
+# FORMAT
+The [TOML format][toml] is used as the encoding of the configuration file. Every option is nested under its table. No bare options are used. The format of TOML can be simplified to:
+
+    [table1]
+    option = value
+
+    [table2]
+    option = value
+
+## CONTAINERS TABLE
+The `containers` table contains settings pertaining to the OCI runtime used and options for how to set up and manage the OCI runtime.
+
+**default_ulimits**=[]
+  A list of ulimits to be set in containers by default, specified as "<ulimit name>=<soft limit>:<hard limit>", for example:"nofile=1024:2048".
+
+**conmon_env**=["PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"]
+  Environment variable list for the conmon process, used for passing necessary environment variables to conmon or the runtime.
+
+**selinux**=false
+  If true, SELinux will be used for pod separation on the host.
+
+**seccomp_profile**="container-default"
+  Path to the seccomp.json profile which is used as the default seccomp profile for the runtime.
+
+**apparmor_profile**=""
+  Used to change the name of the default AppArmor profile of container engines. The default profile name is "container-default".
+
+**cgroup_manager**="cgroupfs"
+ Cgroup management implementation used for the runtime.
+
+**default_capabilities**=[]
+  List of default capabilities for containers. If it is empty or commented out, only the capabilities defined in the containers json file by the user/kube will be added.
+
+  The default list is:
+```
+  default_capabilities = [
+          "AUDIT_WRITE",
+	      "CHOWN",
+	      "DAC_OVERRIDE",
+	      "FOWNER",
+	      "FSETID",
+	      "KILL",
+	      "MKNOD",
+	      "NET_BIND_SERVICE",
+	      "NET_RAW",
+	      "NET_RAW",
+	      "SETGID",
+	      "SETPCAP",
+	      "SETUID",
+	      "SYS_CHROOT",
+  ]
+```
+
+**default_sysctls**=[]
+  List of default sysctls. If it is empty or commented out, only the sysctls defined in the container json file by the user/kube will be added.
+
+**additional_devices**=[]
+  List of additional devices. Specified as "<device-on-host>:<device-on-container>:<permissions>", for example: "--additional-devices=/dev/sdc:/dev/xvdc:rwm". If it is empty or commented out, only the devices defined in the container json file by the user/kube will be added.
```

Comment:
Remove:
" If it is empty or commented out, only the devices defined in the container json file by the user/kube will be added."

Reply:
s/container's json/container json/

Reply:
Actually there is no /usr/kube.

How about?

```
List of additional devices. Specified as "<device-on-host>:<device-on-container>:<permissions>", for example: "--additional-devices=/dev/sdc:/dev/xvdc:rwm". ```